rules-tests/Php83/Rector/ClassConst/AddTypeToConstRector/Fixture/skip_when_parent_class_const_defines_type.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Rector\Tests\Php83\Rector\ClassConst\AddTypeToConstRector\Fixture;
+
+use Rector\Tests\Php83\Rector\ClassConst\AddTypeToConstRector\Source\ParentClass;
+
+class SomeClass extends ParentClass
```

Comment:
could you add fixture for not autoloaded parent/interface/trait to be skipped for all constants in the class as maybe custom autoload

Reply:
for example:

```
class SkipExtendsNotAutoloadedClass extends NotAutoloadedClass
```